src/test/java/ru/yandex/diplom_2/order/CreateOrderWithValidDataTests.java

```diff
@@ -0,0 +1,80 @@
+package ru.yandex.diplom_2.order;
+
+import io.restassured.response.ValidatableResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import ru.yandex.diplom_2.user.User;
+import ru.yandex.diplom_2.user.UserClient;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CreateOrderWithValidDataTests {
+
+    private static UserClient userClient;
+    String token;
+
+    HashMap<String, List> orderHash;
+    List<String> ingredients = new ArrayList<>();
+    OrderClient orderClient;
+    private int expectedStatus = 200;
```

Comment:
статус коды можно было взять из библиотеки:
import  static  org.apache.http.HttpStatus.*;

statusCode(SC_CREATED)

Reply:
I didn't use HttpStatus lib as of now because I like code statuses written straight in the code better. But if it's an issue, I can change it in the next 'fixes' iteration